crud/select/plan.lua

```diff
@@ -182,6 +182,18 @@ function select_plan.new(space, conditions, opts)
         return nil, err
     end
 
+    -- We duplicate this condition so that it works in `filters.gen_func`
+    -- (see function `parse`) - therefore it does not process `conditions`
+    -- (and not converted them to `filtter_conditions` array) with an array
+    -- index equal to opts.scan_condition_num. This condition, together with
+    -- the cobination "inversion" of the scan iterator (see lines below),
+    -- is needed for correct pagination when using a partial index.
```

Comment:
To be honest I don't understand it.

I think that:
```
Since we use pagination we should continue iteration since after tuple.
Such iterator could be run only with index:pairs(key(after_tuple), 'GT/LT').
To preserve original condition we should manually inject it in filter_conditions
(see ...).
```

I think also we can't use "inversion" term for iterator. Inversion is transformation LT -> GT, GE -> LE, but here it is changed just to continue iterator in direct and reverse order.